package.json

```diff
@@ -26,10 +26,9 @@
   },
   "homepage": "https://github.com/agco/passport-openam",
   "dependencies": {
-    "MD5": "^1.2.1",
-    "bluebird": "^2.9.24",
+    "bluebird": "^3.3.4",
     "http-as-promised": "^1.0.0",
-    "lodash": "^3.7.0",
+    "md5": "^2.1.0",
```

Comment:
lodash is not used so it's safe to delete.
The version of md5 we're using has been depreciated. So I've updated to the latest version. I'm trusting that our unit tests have good coverage and will catch any issues.